src/workloads/execution/SSBColumnStoreIndex.yml

```diff
@@ -0,0 +1,542 @@
+              ]
+            }
+          }
+        }
+      },
+      {"$addFields": {"order_year": "$_id.order_year", "customer_nation": "$_id.c_nation"} }
+    ]
+
+Q4_3Query: &Q4_3Query
+  Pipeline:
+    [
+      { $limit: 10000 },
+      {
+        "$match": {
+          "part.p_mfgr": "Manufacturer#1",
+          "order.o_orderdate": {
+            "$gte": ISODate("1997-01-01T00:00:00.000Z"),
+            "$lt": ISODate("1999-01-01T00:00:00.000Z")
+          }
+        }
+      },
+      {
+        "$lookup": {
+          "from": "supplier",
+          "localField": "l_suppkey",
+          "foreignField": "s_suppkey",
+          "as": "supplier"
+        }
+      },
+      {"$addFields": {"supplier": {"$first": "$supplier"}}},
+      {"$match": {"supplier.nation.n_name": "UNITED STATES"}},
+      {
+        "$lookup": {
+          "from": "customer",
+          "localField": "order.o_custkey",
+          "foreignField": "c_custkey",
+          "as": "customer"
+        }
+      },
+      {"$addFields": {"customer": {"$first": "$customer"}}},
+      {"$match": {"customer.nation.region.r_name": "AMERICA"}},
+      {
+        "$group": {
+          "_id": {
+            "order_year": {"$year": "$order.o_orderdate"},
+            "c_nation": "$customer.nation.n_name",
+            "p_brand": "$part.p_brand"
+          },
+          "revenue": {
+            "$sum": {
+              "$subtract": [
+                "$l_extendedprice",
+                {"$multiply": ["$l_extendedprice", "$l_discount"]}
+              ]
+            }
+          }
+        }
+      },
+      {
+        "$addFields": {
+          "order_year": "$_id.order_year",
+          "customer_nation": "$_id.c_nation",
+          "part_brand": "$_id.p_brand"
+        }
+      },
+      {"$sort": {"order_year": 1, "customer_nation": 1, "part_brand": 1}}
+    ]
+
+# Defines test operations.
+TestOperations: &TestOperations
+- OperationMetricsName: Q1_1Query
+  OperationName: aggregate
+  OperationCommand: *Q1_1Query
+- OperationMetricsName: Q1_2Query
+  OperationName: aggregate
+  OperationCommand: *Q1_2Query
+- OperationMetricsName: Q1_3Query
+  OperationName: aggregate
+  OperationCommand: *Q1_3Query
+- OperationMetricsName: Q2_1Query
+  OperationName: aggregate
+  OperationCommand: *Q2_1Query
+- OperationMetricsName: Q2_2Query
+  OperationName: aggregate
+  OperationCommand: *Q2_2Query
+- OperationMetricsName: Q2_3Query
+  OperationName: aggregate
+  OperationCommand: *Q2_3Query
+- OperationMetricsName: Q3Query
+  OperationName: aggregate
+  OperationCommand: *Q3Query
+- OperationMetricsName: Q4_1Query
+  OperationName: aggregate
+  OperationCommand: *Q4_1Query
+- OperationMetricsName: Q4_2Query
+  OperationName: aggregate
+  OperationCommand: *Q4_2Query
+- OperationMetricsName: Q4_3Query
+  OperationName: aggregate
+  OperationCommand: *Q4_3Query
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 1
```

Comment:
Should you should run each test more than once? I found that there is a large degree of difference between the first and last run. 

Have you performed any analysis of how noisy this workload is likely to be? For workloads for more mature features we would like there to be limited noise, but I suspect that this may not be the case for this feature.

Reply:
That is a good question. I'm running an evergreen build now that runs the tests 3 times to see how variable our numbers are: https://spruce.mongodb.com/version/637527a857e85a3432f91829/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

Reply:
It seems like there's a fair amount of variability between the p99 and p50, so I'll run them 3 times instead of once. Addressed in a644af8c93905304760009ee145437dc211f6771 .

Reply:
We would normally recommend running the task that you expect to deploy 3 to 5 times and then comparing the the results:

  
![image](https://user-images.githubusercontent.com/22506/202694079-d8e0f1df-791c-4c07-b907-324a77a5916f.png)


That is, compare the results from different execution numbers. Re-running the patch tasks means that you don't have to compile it every time.